.github/workflows/terragrunt-plan-production.yml

```diff
@@ -24,6 +24,9 @@ env:
   TF_VAR_rds_db_password: ${{ secrets.PRODUCTION_DB_PASSWORD }}
   TF_VAR_slack_webhook: ${{ secrets.PRODUCTION_SLACK_WEBHOOK }}
   TF_VAR_gc_notify_callback_bearer_token: ${{ secrets.PRODUCTION_GC_NOTIFY_CALLBACK_BEARER_TOKEN }}
+  GC_NOTIFY_URL: https://api.notification.canada.ca
```

Comment:
You have removed this variable from the form_viewer json yet its [present in the app](https://github.com/cds-snc/platform-forms-client/pull/720/files#diff-d7a8bdf22aca2702cce5220e6fcd5d284141305db1a909814878479f51e2ffcbR102)